### PR TITLE
feat(feg): base accounting heartbeat feature

### PR DESCRIPTION
**Pull request by emakeev**
Signed-off-by: Evgeniy Makeev <evgeniym@fb.com>

## Summary

Add idle AP accounting heartbeat to update the smart contract accounting service when AP doesn't serve any clients, but up and ready.

## Test Plan
unit tests; test on a partner setup later

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: feg/gateway/services/aaa/base_acct/heartbeat_reset_go1_15.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//go:build go1.15
+// +build go1.15
```
Comment (uri200) [inline feg/gateway/services/aaa/base_acct/heartbeat_reset_go1_15.go:14]: does it mean that package `base_acct` is going to be built with go.1.15?

File: feg/gateway/services/aaa/base_acct/heartbeat_reset_prego1_15.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+// LICENSE file in the root directory of this source tree.
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//go:build !go1.15
+// +build !go1.15
```
Comment (uri200) [inline feg/gateway/services/aaa/base_acct/heartbeat_reset_prego1_15.go:14]: what is this needed?  You create two binaries?

**Review by uri200 — APPROVED** [review]

File: feg/gateway/services/aaa/base_acct/heartbeat_reset_go1_15.go (hunk 1 above)
Comment (emakeev) [reply feg/gateway/services/aaa/base_acct/heartbeat_reset_go1_15.go:14]: It just ticker reset capability, it didn't exist prior to Go 1.15, so the reset call for older compilers will be noop

File: feg/gateway/services/aaa/base_acct/heartbeat_reset_prego1_15.go (hunk 2 above)
Comment (emakeev) [reply feg/gateway/services/aaa/base_acct/heartbeat_reset_prego1_15.go:14]: one is "+build go1.15" -> Go 1.15 or newer & one is "+build !go1.15" -> older then Go 1.15

**Pull request merged**